pkg/api/auth/token/jwt.go

```diff
@@ -13,8 +12,9 @@ const DefaultIssuerName = "kubesphere"
 var errInvalidToken = errors.New("invalid token")
 
 type claims struct {
-	Username string `json:"username"`
-	UID      string `json:"uid"`
+	Username string   `json:"username"`
+	UID      string   `json:"uid"`
+	Groups   []string `json:"groups"`
```

Comment:
If we issue a token include groups, what will happen to the issued token if user groups changed?

Reply:
TokenReview API needs to return group information, it's unnecessary.